packages/vscode-extension/src/commands/ImportObjects.ts

```diff
@@ -131,14 +131,13 @@ export default class ImportObjects extends BaseAction {
 
 		if (objectsFolderAbsolutePaths.length > 0) {
 			const srcFolderAbsolutePath = path.join(rootWorkspaceFolder, SRC_FOLDER_NAME);
-			const transformAbsolutePathsToRelativePathsFunc = (absolutePath: string) => (
-				absolutePath.replace(srcFolderAbsolutePath, '').replace(/\\/g, '/')
-			);
-			const objectsFolderRelativePaths = objectsFolderAbsolutePaths.map(transformAbsolutePathsToRelativePathsFunc)
+			const transformAbsolutePathsToRelativePathsFunc = (absolutePath: string) =>
```

Comment:
I am not sure, but maybe this function could be replaced with path.relative native function: https://nodejs.org/docs/latest-v14.x/api/path.html#path_path_relative_from_to

Reply:
I needed to change line 95 for a change related again with `TranslationService.getMessage` returning a string type . This line has changed due to the formatting but I haven't actually worked in any logical change in this file.
I would prefer to not risk any possible logical change without proper QA testing. I say so, because we are removing some relative path, but also replacing the `\` character in the function.